docs/ref/contrib/gis/install/postgis.txt

```diff
@@ -36,9 +36,7 @@ functionality::
     $ psql <db name>
     > CREATE EXTENSION postgis;
 
-The database user must be a superuser in order to run
-``CREATE EXTENSION postgis;``. The command is run during the :djadmin:`migrate`
-process. An alternative is to use a migration operation in your project::
+The database user must be a superuser in order to run ``CREATE EXTENSION postgis;`` which is run during the :djadmin:`migrate` process. However if the extension already exists it wont create it during this process, hence the database user does not need superuser status to proceed. An alternative is to use a migration operation in your project::
```

Comment:
The line wrapping at 79 characters should be maintained, rather than putting everything on a single line. 

I'm not sure if the migration operation approach is useful. How could `migrate` get to the point of running the migration without the postgis extension being created? (`connection.prepare_database()` runs first).

Reply:
Yes I actually took some time to fix my broken vundle install so that flake8 works again. 

I'd agree with that, I think its not very useful - but I still know to little about the implications it would have to remove it since im so new to the codebase. But if you'd like it removed, ill do it.

Reply:
@claudep can double check my thinking (originally added in 6b54504e).  It's not really related to this change and so could be removed separately. You can leave it for now.

Reply:
Alright, one more question - where in `tests/` could I look for inspiration for this case? and/or where would it be appropriate to put this test?